docs/networking/calico.md

```diff
@@ -106,8 +133,6 @@ For more details on enabling the eBPF dataplane please refer the [Calico Docs](h
   networking:
     calico:
       bpfEnabled: true
-      bpfExternalServiceMode: Tunnel
-      bpfLogLevel: Info
```

Comment:
Any reason to remove these? They were meant to show that there are some optional parameters there.

Reply:
I thought it was confusing, as they're not influenced by the choose of cross-subnet. Could we show these in the section on the eBPF dataplane instead?

Reply:
Thought this was the eBPF section.

Reply:
Could we comment them, to indicate that "Tunnel" is the default, but "Info" is not?